posthog/cdp/templates/microsoft_teams/template_microsoft_teams.py

```diff
@@ -0,0 +1,66 @@
+from posthog.cdp.templates.hog_function_template import HogFunctionTemplate, HogFunctionSubTemplate, SUB_TEMPLATE_COMMON
+
+template: HogFunctionTemplate = HogFunctionTemplate(
+    status="free",
+    id="template-microsoft-teams",
+    name="Microsoft Teams",
+    description="Sends a message to a Microsoft Teams channel",
+    icon_url="/static/services/microsoft-teams.png",
+    category=["Customer Success"],
+    hog="""
+if (not match(inputs.webhookUrl, '^https://[^/]+.webhook.office.com/webhookb2/.*')) {
+    throw Error('Invalid url');
```

Comment:
We'll save a few support pingpongs if in the error we also say what's the right format.

Reply:
added the correct format into the error message